src/sentry/static/sentry/app/views/settings/project/projectProcessingIssues.tsx

```diff
@@ -199,50 +211,55 @@ class ProjectProcessingIssues extends React.Component {
     });
   };
 
-  renderDebugTable = () => {
-    let body;
+  renderDebugTable() {
+    let body: React.ReactNode;
     if (this.state.loading) {
       body = this.renderLoading();
     } else if (this.state.error) {
       body = <LoadingError onRetry={this.fetchData} />;
     } else if (
-      this.state.processingIssues.hasIssues ||
-      this.state.processingIssues.resolveableIssues ||
-      this.state.processingIssues.issuesProcessing
+      this.state.processingIssues &&
```

Comment:
I think if we use `destructuring` here it would be less verbose..but it's just a suggestion